### DISCUSSION TITLE
Line of panel supported? Is there a list of panel that is supported or known to be good with that setup? Like is the powerserie Neo panel are also supported?

### DISCUSSION THREAD

**Discussion by petem001** [Dilbert66/esphome-dsckeybus#156] [General]
Like is the powerseries Neo panel are also supported?

I need to change my old pc5020 (power 8-32) that is dead, and want to integrate the new panel with an esp32. Just love the way you designed the whole thing. Also, since this act as a keypad, do we have to program a keypad in the panel or it acts as the first keypad connected?

**Comment by Dilbert66** [comment]
The NEO panel series are not supported as they use an encrypted bus.  The only series supported are the PC1832,PC1864 or Power832  and perhaps the PC1616.

**Comment by Dilbert66** [comment]
To answer your other question, you do not need to program another keypad. It just acts like an additional keypad automatically.
